.github/workflows/build-publish-pypackage.yml

```diff
@@ -0,0 +1,100 @@
+name: Build and Publish sdist and wheels on pypi
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.*'
+    branches:
+      - main
+      - master
+      - feat/build_wheel
```

Comment:
New PR to remove this?

Reply:
Well, just appended it.
Building when pushing to master/main is arguable, but BBP/NSE conventions seem to suggest it.